.github/workflows/build-docs.yml

```
@@ -103,7 +110,9 @@ jobs:
 
   publish:
     needs: [build]
-    if: (startsWith(github.ref, 'refs/tags/v') && !contains(github.ref, 'rc')) || github.ref == 'refs/tags/docs-publish'
+    if: |
```

actionlint (1.7.7)

context "steps" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)